frontend/components/httpsUrlInput.js

```diff
@@ -13,7 +13,7 @@ const HttpsUrlInput = ({ placeholder, ...props }) => (
   <input
     {...props}
     type="url"
-    pattern="https://.+"
+    pattern="https://.+\.\w{2,}"
```

Comment:
I don't think we can support paths in the CDN, ever. Someone would have to have a server pulling from Federalist, which isn't compliant. (so no paths is ok)